docs/tools/afs.md

```diff
@@ -8,25 +8,30 @@ todo:
 The `afs` command performs some basic actions on a [AFS](../afs/index.md) drive:
 - list the content of an AFS folder
 - archive/restore an AFS content from/to a directory for backup purposes
+- handle (remove/list ...) an AFS inconsistent nodes for cleaning purposes.
```

Comment:
not very clear, I am not sure I understand what you mean. Do you mean that it allows to take corrective actions towards inconsistent nodes?